<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sauvegarde</class>
 <widget class="QDialog" name="Sauvegarde">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sauvegarde</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../ressources/logo.png</normaloff>../ressources/logo.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>82</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="2">
     <widget class="QPushButton" name="selectionFichierBouton">
      <property name="text">
       <string>Parcourir</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelTitre">
      <property name="text">
       <string>Sauvegarde des comptes</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="cheminLineEdit">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>../Backups/</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelChemin">
      <property name="text">
       <string>Dossier</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="fichierLineEdit">
      <property name="placeholderText">
       <string>default_backup</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelFichier">
      <property name="text">
       <string>Nom du fichier</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="extensionComboBox">
      <item>
       <property name="text">
        <string>.business</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>.xml</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tous les fichiers</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="validationFichierBouton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>193</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
